server.js

```diff
@@ -55,208 +56,229 @@ function showHomepage(req, res) {
           //then render the page
           res.render('pages/index', { cities: results.rows, artists: artistsResult.rows });
         })
-        .catch(error => handleErrors(error,res));
+        .catch(error => handleErrors(error, res));
     });
 }
 
 function showArtwork(req, res) {
-  let sql = `SELECT * FROM artworks WHERE artist=$1;`;
+  let sql = `SELECT  FROM artworks WHERE artist=$1;`;
   let values = [req.params.name];
   console.log(req.params.name);
   client.query(sql, values)
     .then(artworksResults => {
       console.log(artworksResults.rows);
       res.render('pages/savedArtist', { artworks: artworksResults.rows });
     })
-    .catch(error => handleErrors(error,res));
+    .catch(error => handleErrors(error, res));
 
 
 };
-function deleteArtists(request,response) {
-  console.log('request.body',request.params.artistName);
+function deleteArtists(request, response) {
+  console.log('request.body', request.params.artistName);
   let artistName = request.params.artistName;
-  const SQL = 'DELETE FROM artworks WHERE artist=$1;'
+  const SQL = 'DELETE FROM artworks WHERE artist=$1;';
   const VALUES = [artistName];
-  client.query(SQL,VALUES)
-    .then( ()=> {
-    
+  client.query(SQL, VALUES)
+    .then(() => {
+
       response.status(200).redirect('/');
     })
-    .catch( error => {
+    .catch(error => {
       console.error(error.message);
     });
-  };
+}
 
+function showMuseums(req, res) {
+  let sql = `SELECT museum, COUNT (*) AS totalartworks FROM artworks WHERE city=$1 GROUP BY museum ORDER BY totalartworks DESC;`;
+  let values = [req.query.name];
+  console.log(values);
+  client.query(sql, values)
+    .then(results => {
+      console.log(results);
+    });
+}
+// let sql2 = `SELECT city, COUNT(*) AS totalartworks FROM artworks GROUP BY city ORDER BY totalartworks DESC`;//referebce sql statement
 
 function getArtworkResults(req, res) {
   //get the term the user searched for
   let artist = req.body.search;
+  let sql1 = `SELECT * FROM artworks WHERE artist=$1;`;
+  client.query(sql1, [artist])
+    .then(result => {
+      if (result.rowCount) {
+        console.log('already exists');
+        res.status(200).send;
```

Comment:
should redirect them to that saved artist's page that we already have.

Reply:
the res.render is rendering to the savedartist page.